lemarche/crm/management/commands/crm_brevo_sync_companies.py

```diff
@@ -34,11 +34,25 @@ def handle(self, recently_updated: bool, **options):
         self.stdout.write(f"Sync Siae > Brevo: we have {Siae.objects.count()} siaes")
         # Update only the recently updated siaes
         if recently_updated:
-            siaes_qs = siaes_qs.filter(updated_at__gte=ten_days_ago)
+            siaes_qs = siaes_qs.filter(updated_at__gte=two_weeks_ago)
             self.stdout.write(f"Sync Siae > Brevo: {siaes_qs.count()} recently updated")
 
-        # Step 2: loop on the siaes
+        # Step 2: Add the 90-day limited annotations
+        siaes_qs = Siae.objects.with_tender_stats(days=90)
```

Comment:
Je pense qu'ici, tu dois continuer avec le même query_set :
```suggestion
        siaes_qs = siaes_qs.with_tender_stats(days=90)
```